README.md

```diff
@@ -379,6 +379,12 @@ CPMAddPackage(
 )
 ```
 
+### [simdjson](https://github.com/simdjson/simdjson)
+
+```
+CPMAddPackage("gh:simdjson/simdjson@3.5.0")
+```
+
```

Comment:
```suggestion
```

To avoid clutter I would omit this example from the main readme. My intention with examples in the main readme is to showcase different levels of complexity when adding popular libraries, and we already have a project that works well with the single-argument syntax and `@` version specifier. From this users should be able to easily figure out how to add `simdjson` in addition to the provided example project.